generate_model.py

```diff
@@ -0,0 +1,29 @@
+import os
+import os.path
+
+model_template = """
+module {module_name}.Model where
+
+type alias Model =
+  {{}}
+"""
+
+def make_boilerplate(module_name, directory):
```

Comment:
We can probably reuse this for the other files if we take the `template` and `file_name` as arguments